sql_generators/mobile_kpi_support_metrics/templates/new_profiles.schema.yaml

```diff
@@ -0,0 +1,65 @@
+fields:
+- mode: NULLABLE
+  name: first_seen_date
+  type: DATE
+  description: When the profile has been seen for the first time.
+
+- mode: NULLABLE
+  name: normalized_channel
+  type: STRING
+  description: Release channel of the app the profile is using.
+
+- name: app_name
+  type: STRING
+  mode: NULLABLE
+  description: App name the profile is using.
+
+- name: app_version
+  type: STRING
+  mode: NULLABLE
+  description: Client's app version on the metric date.
```

Comment:
```suggestion
  description: Client's app version on the first_seen_date.
```

Reply:
Never mind, it looks like this is an aggregate. In this case it would be beneficial to name it new_profiles_aggregates for consistency with the convention.

Reply:
this is consistent with the aggregate table names within the same generator, i.e. `retention` and `engagement`.

Reply:
I see, shall  we align those too for a consistent naming in the DWH?

Reply:
This is something we agreed on when initially creating the retention and engagement modeling. I'd suggest if you think we should change this that we have a larger conversation as a team about this. However, I believe this falls outside of the scope of this change.